wordpress-mobile/WordPress-iOS
# Swifting WPNUXHelpBadgeLabel

### Details:
Analog to #8579, in this small PR we're porting `WPNUXHelpBadgeLabel` over to Swift!

### To test:
1. Hack [this spot](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/NUX/NUXViewControllerBase.swift#L161) and paste the following snippet:

```
helpBadge.isHidden = false
helpBadge.text = "1"
```

2. Just in case, comment [the body of this method](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/NUX/NUXViewControllerBase.swift#L100)

3. Fresh install the app, and press the `Login` button

Verify that a red bubble shows up on top of the Help button.

Needs Review: @frosty 
Thaaaanks in advance!!
